test/test_rigidbody.cpp

```diff
@@ -269,6 +269,18 @@ TEST(Dynamics, ForwardAccelerationConstraint){
         EXPECT_NEAR(cs.force[i], forces_expected[i], requiredPrecision);
 }
 
+TEST(Kinematics, computeQdot)
+{
+    biorbd::Model m("models/simple_quat.bioMod");
+    biorbd::rigidbody::GeneralizedCoordinates Q_quat(m.nbQ()), QDot(m), QDot_quat(m.nbQ()), QDot_quat_expected(m.nbQ());
+    Q_quat << 0,0,0,1;
```

Comment:
Is there a risk of false positive in using the unitary quaternion?

Reply:
I don't think so, but I'll change that in order to be 100% sure